rules/relatives_ibis.smk

```diff
@@ -40,44 +72,78 @@ def aggregate_input(wildcards):
     return expand(f"ibd/{{id}}.tsv", id=ids)
 
 
-rule ersa:
-    input:
-        ibd=aggregate_input
-    output:
-        "ersa/relatives.tsv"
-    conda:
-        "../envs/ersa.yaml"
-    log:
-        "logs/ersa/ersa.log"
-    benchmark:
-        "benchmarks/ersa/ersa.txt"
-    params:
-        ersa_l = config['zero_seg_count'],
-        ersa_th = config['zero_seg_len'],
-        alpha = config['alpha'],
-        ersa_t = config['ibis_seg_len']
-    shell:
-        """
-        ERSA_L={params.ersa_l} # the average number of IBD segments in population
-        ERSA_TH={params.ersa_th} # the average length of IBD segment in population
-        ERSA_T={params.ersa_t} # min length of segment to be considered in segment aggregation
-        
-        FILES="{input.ibd}"
-        TEMPFILE=ersa/temp_relatives.tsv
-        rm -f $TEMPFILE
-        rm -f {output}
-        
-        for input_file in $FILES; do
-
-            ersa --avuncular-adj -ci --alpha {params.alpha} --dmax 14 -t $ERSA_T -l $ERSA_L -th $ERSA_TH $input_file -o $TEMPFILE  |& tee {log}
-
-            if [[ "$input_file" == "${{FILES[0]}}" ]]; then
-                cat $TEMPFILE >> {output}
-            else
-                sed 1d $TEMPFILE >> {output}
-            fi
-        done
-        """
+if config['weight_mask']:
```

Comment:
One can do 
params: 
  mask_argument = '--nomask' if config['weight_mask'] else ''
and `ersa {params.mask_argument} --avuncular-adj ...
It will be easier to see what changed in ersa invocation this way and also more compact